python/conda_install.sh

```diff
@@ -33,4 +33,5 @@ export USER_BOOST_INCLUDE="-I $BASE_BOOST/include/boost -I $BASE_BOOST/include"
 export USER_BOOST_LIBRARY="-L $BASE_BOOST/lib"
 
 # build the C++ binaries and python package
-python python/setup.py install
+python vowpal_wabbit/python/setup.py install
+pip install -U vowpal_wabbit/python/
```

Comment:
That is something i do not understand either. But without it there is a local package build as well as `vowpalwabbit-8.5.0-py3.6-linux-x86_64.egg` among conda packages, but the package itself is not installed into the the conda packages. Thus, `import vowpalwabbit` fails since it can not find the package.

Reply:
That works to some extend, I have tried that as well:) But it doesn't install the package inside the conda environment. Instead, it keeps the build locally and sets an egg-link, if I remember correctly. This allows to import the package. But I'm not sure how robust is it for all possible use cases, i.e. wrapping in into a docker image or whatever else people might want to do with it. I have close-to-zero experience with docker, so I might just misunderstand how it functions. If you think that is it good-enough solution, than I can switch to that